charts/aws-ebs-csi-driver/templates/tests/helm-tester.yaml

```diff
@@ -214,6 +216,13 @@ spec:
       volumeMounts:
       - name: config-vol
         mountPath: /etc/config
+      resources:
+        requests:
+          cpu: "100m"
+          memory: "128Mi"
+        limits:
+          cpu: "200m"
```

Comment:
We shouldn't use a CPU limit, even on test pods: https://home.robusta.dev/blog/stop-using-cpu-limits

otherwise lgtm (might need to bump up the RAM limit? not sure if it's enough when running the tests in parallel)

Reply:
Good call on the CPU limits. Will run some tests offline to pick a more suitable memory limit

Reply:
New values picked from several testing runs:

```
kubectl top pod ebs-csi-driver-test -n kube-system

NAME                  CPU(cores)   MEMORY(bytes)
ebs-csi-driver-test   1418m        2852Mi
```